README.md

```diff
@@ -35,6 +35,9 @@ To determine whether to allow response _response_ to a request _request_, run th
    1. Append (_request_'s opaque media identifier, _request_'s current URL) to the user agent's opaque-safelisted requesters set.
    1. Return true.
 1. If _nosniff_ is true, then return false.
+1. If _response_'s status is not an [ok status](https://fetch.spec.whatwg.org/#ok-status), then return false.
+1. If _mimeType_'s essence starts with "`audio/`", "`image/`", or "`video/`", then return false.
```

Comment:
Would it make sense to use case-insensitive comparison here?  Looking at httpArchive, I see that the content types returned in the wild sometimes includes things like `IMAGE/PNG`.

Also - would it make sense to shorten the required prefix to just `image`?  Again, looking at httpArchive, I see that the content types returned in the wild sometimes includes things like `images/gif` and `image\\gif'.

Reply:
They are already lowercased when we parse the MIME type. `image\gif` would result in _mimeType` being failure. `images/gif` might be interesting to include, but we should then add it to Fetch as well and block it in general (also for same-origin requests).

Reply:
Ack.  Sounds good.